components/src/core/drawer/utilities.tsx

```diff
@@ -11,8 +11,11 @@ import { AllSharedProps } from './types';
  * @param childValue The value of the property on the child
  * @returns The child value if it is defined, otherwise the parent value
  */
+
+/* eslint-disable */
 export const mergeStyleProp = <T extends unknown>(parentValue: T, childValue: T): T | undefined =>
```

Comment:
Why is eslint complaining about this function?

Reply:
welp... I was wrong. 

```
15:32  error  Constraining the generic type 'T' to 'unknown' does nothing and is unnecessary  @typescript-eslint/no-unnecessary-type-constraint

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
```

Reply:
If you wanted to, you could just take off the `extends unknown` part. But either way is fine.